librocksdb-sys/build.rs

```diff
@@ -26,9 +26,16 @@ fn fail_on_empty_directory(name: &str) {
     }
 }
 
+fn rocksdb_include_dir() -> std::string::String {
```

Comment:
Why is it needed? Do you have a specific use-case.

Reply:
I would like to build the `rocksdb` crate with a system-provided `rocksdb` library. So in my build recipe I'm setting:

`ROCKSDB_INCLUDE_DIR = "${rocksdb}/include";`

where `rocksdb` is the path to the `rocksdb` on my system.

Reply:
By the way. A system-provided library places its headers with paths like `/usr/include` in linux case or `/usr/local/include` in macos case. A compiler uses these paths by default AFAIK 🤔 This var could be useful when you use sources from other different places.

Reply:
Indeed. In my case I set:
```
ROCKSDB_INCLUDE_DIR=/nix/store/m9xx9jq9b6vz42zy8xdzj03grskdlm89-rocksdb-6.6.4/include
```
since I use the Nix build system.

Reply:
Yes. I think it could be useful in some cases.